cmd/otel-allocator/main.go

```diff
@@ -189,6 +199,20 @@ func main() {
 				setupLog.Error(shutdownErr, "Error on server shutdown")
 			}
 		})
+	if cfg.HTTPS.Enabled {
```

Comment:
do we need to put the above srv.Start() in an else block or are we okay to potentially run both a secure and insecure simultaneously? I guess doing it this way would make for a better migration from http to https.

Reply:
Based on previous conversation, it was suggested to have both HTTP and HTTPS running at the same time.